htdocs/product/class/product.class.php

```diff
@@ -3157,6 +3157,17 @@ function getNomUrl($withpicto=0,$option='',$maxlength=0)
 
         $linkclose = '" title="'.str_replace('\n', '', dol_escape_htmltag($label, 1)).'" class="classfortooltip">';
 
+	if (! is_object($hookmanager))
```

Comment:
This test will always be true because hookmanager seems to not be imported with global $hookmanager